src/test/java/com/google/sps/data/ScheduledInterviewServletTest.java

```diff
@@ -54,184 +54,371 @@
   LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalCapabilitiesServiceTestConfig());
   private FakeScheduledInterviewDao scheduledInterviewDao;
+  private FakeAvailabilityDao availabilityDao;
   private FakePersonDao personDao;
 
-  @Before
-  public void setUp() {
-    helper.setUp();
-    scheduledInterviewDao = new FakeScheduledInterviewDao();
-    personDao = new FakePersonDao();
-  }
-
-  @After
-  public void tearDown() {
-    helper.tearDown();
-  }
-
-  private final Person interviewer =
+  private final Person GoogleSWE1 =
```

Comment:
Fix variable capitalization (here and below)

Reply:
Done